qiskit/compiler/assembler.py

```diff
@@ -154,3 +162,93 @@ def assemble_circuits(circuits, run_config=None, qobj_header=None, qobj_id=None)
 
     return QasmQobj(qobj_id=qobj_id or str(uuid.uuid4()), config=userconfig,
                     experiments=experiments, header=qobj_header)
+
+
+def _replaced_with_user_los(user_lo_dict, default_los):
+    """Return user LO frequencies replaced with `default_los`.
+    Args:
+        user_lo_dict(UserLoDict): dictionary of user's LO frequencies
+        default_los(list(float)): default LO frequencies to be replaced
+    Returns:
+        List: user LO frequencies
+    """
+    res = copy.copy(default_los)
+    for channel, user_lo in user_lo_dict.items():
+        res[channel.index] = user_lo
+
+    return res
+
+
+def assemble_schedules(schedules, dict_config, dict_header, converter=PulseQobjConverter):
```

Comment:
Why does this need to be supplied with `ConditionedSchedule`s? It seems clunky to require the user to learn another class just for assembly. Could the config not be provided as an extra option?